config/runtime.exs

```diff
@@ -3,17 +3,29 @@ import Config
 # Database
 env = config_env() |> Atom.to_string()
 partition = System.get_env("MIX_TEST_PARTITION")
-default_database_url = "ecto://postgres:postgres@localhost:5432/baz_?"
+default_database_url = "ecto://postgres:postgres@localhost:5431/baz_?"
```

Comment:
Can we keep `5432` :smile: 

You should be able to configure your own local env vars to customize the `DATABASE_URL`

Reply:
Sorry, didn't mean to commit!    😇